### ISSUE TITLE
iOS Crash on Init

### ISSUE THREAD

**Issue by robisaks** [pollfish/flutter-plugin-pollfish#3]
The iOS plugin hard crashes the app when attempting to init the pollfish sdk. Works on Android

My init code
```
 Future<void> initPollfish() async {
    SharedPreferences prefs = await SharedPreferences.getInstance();
    String id = prefs.getString("id");
    FlutterPollfish.instance.init(
      apiKey: 'KEY',
      requestUUID: id,
      rewardMode: true,
      offerwallMode:false,
      releaseMode: true);
    FlutterPollfish.instance.setPollfishReceivedSurveyListener(onPollfishSurveyReveived);
    FlutterPollfish.instance.setPollfishCompletedSurveyListener(onPollfishSurveyCompleted);
    FlutterPollfish.instance.setPollfishSurveyOpenedListener(onPollfishSurveyOpened);
    FlutterPollfish.instance.setPollfishSurveyClosedListener(onPollfishSurveyClosed);
    FlutterPollfish.instance.setPollfishSurveyNotAvailableSurveyListener(onPollfishSurveyNotAvailable);
    FlutterPollfish.instance.setPollfishUserRejectedSurveyListener(onPollfishUserRejectedSurvey);
    FlutterPollfish.instance.setPollfishUserNotEligibleListener(onPollfishUserNotEligible);
  }
```

The crash
```
flutter:  invokeMethod FlutterPollfish.init()...
2019-08-20 09:59:55.288736-0500 Runner[20954:901316] -[NSNull integerValue]: unrecognized selector sent to instance 0x11302ef28
2019-08-20 09:59:55.329920-0500 Runner[20954:901316] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSNull integerValue]: unrecognized selector sent to instance 0x11302ef28'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000112dbc1bb __exceptionPreprocess + 331
	1   libobjc.A.dylib                     0x000000011235a735 objc_exception_throw + 48
	2   CoreFoundation                      0x0000000112ddaf44 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
	3   CoreFoundation                      0x0000000112dc0ed6 ___forwarding___ + 1446
	4   CoreFoundation                      0x0000000112dc2da8 _CF_forwarding_prep_0 + 120
	5   Runner                              0x000000010a3953bf -[FlutterPollfishPlugin callInitialize:result:] + 943
	6   Runner                              0x000000010a394e8f -[FlutterPollfishPlugin handleMethodCall:result:] + 191
	7   Flutter                             0x000000010d40769e __45-[FlutterMethodChannel setMethodCallHandler:]_block_invoke + 115
	8   Flutter                             0x000000010d3a7016 _ZNK7flutter21PlatformMessageRouter21HandlePlatformMessageEN3fml6RefPtrINS_15PlatformMessageEEE + 166
	9   Flutter                             0x000000010d3aa52c _ZN7flutter15PlatformViewIOS21HandlePlatformMessageEN3fml6RefPtrINS_15PlatformMessageEEE + 38
	10  Flutter                             0x000000010d401f07 _ZNSt3__110__function6__funcIZN7flutter5Shell29OnEngineHandlePlatformMessageEN3fml6RefPtrINS2_15PlatformMessageEEEE4$_27NS_9allocatorIS8_EEFvvEEclEv + 57
	11  Flutter                             0x000000010d3b72cb _ZN3fml15MessageLoopImpl10FlushTasksENS_9FlushTypeE + 123
	12  Flutter                             0x000000010d3bbb16 _ZN3fml17MessageLoopDarwin11OnTimerFireEP16__CFRunLoopTimerPS0_ + 26
	13  CoreFoundation                      0x0000000112d21f34 __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 20
	14  CoreFoundation                      0x0000000112d21b32 __CFRunLoopDoTimer + 1026
	15  CoreFoundation                      0x0000000112d2139a __CFRunLoopDoTimers + 266
	16  CoreFoundation                      0x0000000112d1ba1c __CFRunLoopRun + 2252
	17  CoreFoundation                      0x0000000112d1ae11 CFRunLoopRunSpecific + 625
	18  GraphicsServices                    0x000000011524f1dd GSEventRunModal + 62
	19  UIKitCore                           0x000000011ae7781d UIApplicationMain + 140
	20  Runner                              0x000000010a0398f0 main + 112
	21  libdyld.dylib                       0x000000011374a575 start + 1
	22  ???                                 0x0000000000000001 0x0 + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException


```

**Comment by vourkosa** [comment]
@robisaks you are missing two params:

      pollfishPosition: pollfishPosition,
       indPadding: indPadding,

for example you can use:

      pollfishPosition: 0,
       indPadding: 50,

during initialization. If you add them it will work. We will provide a fix in the next update of the plugin.

**Comment by vourkosa** [comment]
This is fixed on v0.2.2
